tracer/test/snapshots/Security.AspNetMvc5AsmRulesToggle.Classic.enableSecurity=False._.verified.txt

```diff
@@ -116,7 +116,7 @@
       http.request.headers.host: localhost:00000,
       http.status_code: 200,
       http.url: http://localhost:00000/health,
-      http.useragent: Mistake Not... acunetix-product,
+      http.useragent: Mistake Not... sql power injector sql power injector sql power injector,
```

Comment:
Is it expected to have the user agent argument repeated?

Reply:
It's definitely weird and I will log a ticket to investigate, but it must be exposing an existing bug rather than create an new one.